api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -51,11 +51,125 @@ message CommunicationProtocol {
   oneof Framing {
     // AX.25 protocol settings.
     AX25 ax25 = 1;
+
+    // CCSDS framing settings.
+    CCSDS ccsds = 2;
+
+    // Bit stream framing settings.
```

Comment:
Oops - we're using `bitstream` in our wording, not `bit stream`. I notice the message that was already there is `BitStream` but luckily it's not used. Can you go ahead and rename it to `Bitstream` and use the one word version everywhere?